lib/l10n-en_GB.js

```diff
@@ -215,18 +215,21 @@ exports.messages = {
     'sotd.pp.no-errata':
         'No errata link was found in the patent policy section.',
     'sotd.pp.no-response': 'The connection to the API failed: ${error}',
-    // sotd/charter-disclosure
-    'sotd.charter-disclosure.no-group': 'Unable to find the deliverer id.',
-    'sotd.charter-disclosure.text-not-found':
+    'sotd.pp.joint-different-pp':
+        'This document is published by multiple groups using different versions of Patent Policy. See requirement details at <a href="https://github.com/w3c/specberus/issues/1319#issuecomment-878619033">this GitHub Issue</a>.',
```

Comment:
I don't think we should link to a comment which can be edited in the future.

```suggestion
        'This document is published by multiple groups operating under different versions of the Patent Policy. A document should be published under one and only one Patent Policy, to which all participants in contributing groups have agreed.',
```